api.raml

```diff
@@ -23,28 +22,79 @@ securitySchemes:
           type: string
 
 
-#
-# Example Endpoints
-#
+/sequenceForGff3/{sequenceType}:
+  displayName: Retrieve sequence for GFF3
```

Comment:
I feel like it would be good for the caller to declare what kind of file they are sending?  But the API for these two endpoints seems to be the same, so a single "endpoint" with a param might be best i.e. /sequences-of/{type}, where type is gff or bed.

Also we've been typically using dash syntax for url segments over camel case when we can.

Reply:
`/sequences-of/{type}` seems very good!